ERCS/erc-9999.md

```diff
@@ -0,0 +1,644 @@
+---
+eip: 9999
```

Comment:
```suggestion
eip: 7845
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.

Reply:
Updated - thank you @abcoathup